templates/email/messages/berth_offer_switch_en.html

```diff
@@ -176,7 +120,7 @@ <h3>Price of the boat berth</h3>
                 1. Read the attached boat berth lease agreement terms and the harbor rules
             </p>
             <p>
-                2. Notify us that you accept the boat berth change offered via this link:
+                2. Notify us that you accept the boat berth change offered via this link, due date is {{ due_date }}:
```

Comment:
Nitpick:
```suggestion
                2. Notify us that you accept the boat berth change offered via this link, the due date is {{ due_date }}:
```
or
```suggestion
                2. Notify us that you accept the boat berth change offered via this link before {{ due_date }}:
```

Reply:
fixed. I opted for "the due date is" because the meaning is "on or before"